Jenkinsfile

```diff
@@ -19,11 +19,42 @@ elifePipeline {
                 sh "docker stop loris"
             }
         }
+    }
+
+    elifeMainlineOnly {
+        stage 'Deploy on ci, continuumtest', {
+            def deployments = [
+                ci: {
+                    lock('iiif--ci') {
+                        builderDeployRevision 'iiif--ci', commit
+                        builderSmokeTests 'iiif--ci', '/opt/loris'
+                    }
+                },
+                continuumtest: {
+                    lock('iiif--continuumtest') {
+                        builderDeployRevision 'iiif--continuumtest', commit
+                        builderSmokeTests 'iiif--continuumtest', '/opt/loris'
+                    }
+                }
+            ]
+            parallel deployments
+        }
+
+        stage 'End2end tests', {
```

Comment:
should this be run each time or only on commits to `master`?

and ... how is the 'mainline' branch defined? this repo doesn't have a 'master' branch...

Reply:
```
vars/elifeMainlineOnly.groovy
2:    // env.BRANCH_NAME is only specified for multibranch projects
5:    // since they have env.BRANCH_NAME
6:    if (env.BRANCH_NAME == null) {
```

essentially, if BRANCH_NAME exists, it's on a non-mainline branch